main.go

```diff
@@ -80,13 +90,23 @@ func executeCheck(event *corev2.Event) (int, error) {
 
 	myPid := os.Getpid()
 	processes, _ := process.Processes()
-	for _, process := range processes {
+	for _, proc := range processes {
 		// Skip myself
-		if process.Pid == int32(myPid) {
+		if proc.Pid == int32(myPid) {
 			continue
 		}
-		name, _ := process.Name()
-		cmdline, _ := process.Cmdline()
+
+		// Check for zombie processes if --zombie or -z flag is set
+		if plugin.Zombie {
+			status, _ := proc.Status()
+			if status == "Z" { // "Z" status is for Zombie processes
+				fmt.Printf("Zombie process found with PID: %d\n", proc.Pid)
```

Comment:
It might be worth listing all the zombie processes before terminating, so that if there are more than one, the output doesn't change between runs